src/Microsoft.OData.Core/ODataReader.cs

```diff
@@ -30,10 +32,38 @@ public abstract class ODataReader
         /// <returns>true if more items were read; otherwise false.</returns>
         public abstract bool Read();
 
+        /// <summary>Creates a stream for reading an inline stream property. </summary>
+        /// <returns>A stream for reading the stream property.</returns>
+        public virtual Stream CreateReadStream()

```

Comment:
>CreateReadStream [](start = 30, length = 16)

Maybe rename it as "CreateStream()"

Reply:
I like CreateReadStream because it is more explicit that the stream is used for reading the value.

---
In reply to: [268359194](https://github.com/OData/odata.net/pull/1204#discussion_r268359194) [](ancestors = 268359194)